content/dstu2/goal.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Goal | DSTU 2 API
+---
+
+# Goal
+
+* TOC
+{:toc}
+
+## Terminology Bindings
+
+<%= terminology_table(:Goal, :dstu2) %>
+
+## Search
+
+Search for Goals that meet supplied query parameters:
+
+    GET /Goal?:parameters
+
+### Parameters
+
+ Name              | Required?                               | Type          | Description
+-------------------|-----------------------------------------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ `_id`             | Yes, or one of `patient`, or `subject`. | [`token`]     | The logical resource id associated with the Goal. Example: `7891`
+ `patient`         | Yes, or one of `_id`, or `subject`.     | [`reference`] | The patient who has the goal. Example: `12345`
+ `subject:Patient` | Yes, or one of `_id`, or `patient`.     | [`reference`] | The subject(Patient) who this goal is intended for. Example: `12345`
+ `targetdate`      | N                                       | [`date`]      | A date or date range from which to find Goals. The `targetDate` parameter may be provided once with a prefix to imply a date range or without a prefix to search for goals at a specific date. Alternately it may be provided twice with `le`, `lt`, `ge`, or `gt` prefixes to search for goals within specific range. The date and prefix pairs must create a closed range. Example: `targetdate=ge2016-10-01&targetdate=le2016-12-01`
```

Comment:
Also, should we move some of this description down to a notes section? It'll be easier to read.